controller.go

```diff
@@ -86,17 +93,13 @@ func (c *K8SController) Start(wg *sync.WaitGroup) {
 		log.Info().Msgf("[%s] Informer is ready and synced", c.id)
 	}
 	go func() {
-		var hasLoopedOnce bool
 		for {
 			item, quit := c.queue.Get()
-			c.mainloop(item)
-			if !hasLoopedOnce {
-				wg.Done()
-				hasLoopedOnce = true
-			}
 			if quit {
+				wg.Done()
 				return
 			}
+			c.mainloop(item)
```

Comment:
`hasLoopedOnce` prevents the controller from processing events. This caused `kubectl opslevel service preview` to always show either one or zero events.

Reply:
The `if quit` statement here on its own will protect against the `nil pointer deref`, but we should keep the check inside `mainloop()` because it takes an `interface{}` which should always be checked for nil.

Reply:
Update: https://github.com/OpsLevel/opslevel-k8s-controller/pull/19